playbooks/group_vars/prometheus/public.yml

```diff
@@ -35,6 +35,9 @@ PROMETHEUS_ALERT_DISK_SPACE_THRESHOLD: 80
 PROMETHEUS_ALERT_HTTP_RESPONSE_TIME_THRESHOLD: 1
 PROMETHEUS_ALERT_HTTP_RESPONSE_FRONTEND_ERROR_THRESHOLD: 70
 
+OCIM_QUEUE_REDIS_KEY: null
+PROMETHEUS_ALERT_OCIM_QUEUE_SIZE_THRESHOLD: null
+
```

Comment:
Can these be defined elsewhere? I'm asking because the rest of this file only has generic stuff for applications which are not specific to OpenCraft like Ocim is.